drivers/sensor/sensirion/scd4x/scd4x.c

```diff
@@ -6,14 +6,18 @@
 
 #define DT_DRV_COMPAT sensirion_scd4x
 
+#include "scd4x.h"
 #include <zephyr/device.h>
 #include <zephyr/drivers/i2c.h>
 #include <zephyr/drivers/sensor.h>
 #include <zephyr/logging/log.h>
 #include <zephyr/sys/crc.h>
```

Comment:
Local includes should come after:

```suggestion
#include <zephyr/device.h>
#include <zephyr/drivers/i2c.h>
#include <zephyr/drivers/sensor.h>
#include <zephyr/logging/log.h>
#include <zephyr/sys/crc.h>

#include "scd4x.h"
```

Reply:
Resolved in [05ab042](https://github.com/catie-aq/zephyr_sensirion-scd4x/pull/11/commits/05ab042a37c44efd45008811e89e4bdc10242298).